frontend/src/routes/dashboard/[groupId]/+page.svelte

```diff
@@ -46,6 +48,9 @@
 		const calendarMode = g.calendarMode.split(' to ');
 		const dateFormat = 'YYYY-MM-DD';
 		console.log(calendarMode);
+		const user = await getUser();
+		currentUserID = user?.userId;
+		console.log('current user id:', currentUserID);
```

Comment:
Have you considered making the model responsible for keeping track of the current user id via a store e.g.
```js
// model.js
export const userId = store(null);

...

getUser().then(user => {
  userId.set(user.userId);
});

// +page.svelte
import {userId} from "$lib/model.js";

console.log($userId);
```

This would avoid lots of redundant calls to `getUser`.